ci-operator/step-registry/openshift/configure-cincinnati/openshift-configure-cincinnati-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: openshift-configure-cincinnati
+  from: cli
+  commands: openshift-configure-cincinnati-commands.sh
+  timeout: 120s
+  resources:
+    requests:
+      cpu: 200m
+      memory: 500Mi
+  env:
+  - name: CINCINNATI_URL
+    default: https://api.openshift.com/api/upgrades_info/v1/graph
+    documentation: "URL of the Cincinnati instance to use."
+  - name: CHANGE_CHANNEL_BASE
+    default: keep
+    documentation: "If a cluster has a channel set, and this parameter is set to one of 'stable', 'fast', 'candidate' or 'eus' then the cluster channel will be switched to the specified channel of the same version. No changes are done otherwise."
+  documentation: |-
+    Configure the cluster to use specified OSUS (Cincinnati) instance and optionally change the channel the cluster is subscribed to.
```

Comment:
nit: Can we say "OpenShift Update Service (OSUS, Cincinnati)" or some such here?  That should get all the keywords in place for greppers, and still spell things out in recognizable words for folks searching the operator name (or who aren't familiar with the more niche OSUS/Cincinnati phrasing).